client/reader/site-subscriptions-manager/reader-site-subscriptions.tsx

```diff
@@ -20,6 +18,11 @@ import NotFoundSiteSubscriptions from './not-found-site-subscriptions';
 
 const SEARCH_KEY = 's';
 
+function getUrlQuerySearchTerm(): string {
+	const queryArgs = getQueryArgs( window.location.href );
+	return ( queryArgs[ SEARCH_KEY ] as string ) ?? '';
+}
```

Comment:
Function that reads the `?s=` query param from URL, in such a shape that it can be used directly as `useState` initializer callback.